### PR TITLE
Chase aabb

### PR DESCRIPTION
**Pull request by m3t4f1v3**
## What this PR does:
- [ ] Bug fix 
- [x] Feature
- [ ] Code refactor / improvement
- [ ] API addition / improvement
- [x] Compat with another mod

**Explain what this PR is doing:**
interpolates create chase aabb between ship position and rotation

---

## Modloaders this PR affects:
- [x] Forge
- [x] Fabric

## Where this PR was tested:
- [x] In-dev singleplayer
- [ ] In-dev dedicated server
- [ ] Out of dev singleplayer
- [ ] GameTest framework

## Breaking Changes
- [ ] Yes (describe)
- [x] No

---

## Additional Notes
Anything else reviewers might need to know:
Performance concerns, edge cases, etc
videos in offtopic
---

## Declaration
By submitting this PR, I affirm:  
- Code/data compiles and loads  
- Tests _(if any)_ pass  
- This PR will not brick worlds

### CODE REVIEW

**Comment by m3t4f1v3** [comment]
based upon #1897 btw, works without though

File: common/src/main/java/org/valkyrienskies/mod/mixin/mod_compat/create/client/MixinChasingAABBOutline.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        method = "render",
+        at = @At(
+            value = "INVOKE",
+            target = "Lnet/createmod/catnip/outliner/ChasingAABBOutline;renderBox(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/createmod/catnip/render/SuperRenderTypeBuffer;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/phys/AABB;Lorg/joml/Vector4f;IZ)V"
+        )
+    )
+    private void interpolatedPose(
+        PoseStack ms, SuperRenderTypeBuffer buffer, Vec3 camera, float pt, CallbackInfo ci
+    ) {
+        ms.pushPose();
```
Comment (Tribulla) [inline common/src/main/java/org/valkyrienskies/mod/mixin/mod_compat/create/client/MixinChasingAABBOutline.java:78]: This can cause a PoseStack Underflow Crash
